src/world/climate.h

```diff
@@ -26,20 +26,22 @@ enum {
 	CLIMATE_COLD
 };
 
-enum{
+enum {
 	WEATHER_SUNNY,
 	WEATHER_PARTIALLY_CLOUDY,
 	WEATHER_CLOUDY,
 	WEATHER_RAIN,
 	WEATHER_HEAVY_RAIN,
-	WEATHER_THUNDER
+	WEATHER_THUNDER,
+	WEATHER_SNOW
```

Comment:
1. Whilst it is snowing, save the game.
2. Load that saved game in vanilla RCT2.
3. Does it crash?

Yes Vanilla RCT2 didn't implement snow which is why you need to check what happens. We still support it and we can't let new features caused saved games to not work in vanilla.